flow-typed/submissions.js

```diff
@@ -1,13 +1,18 @@
 // @flow
+// @ts-nocheck
 
 declare type NewDraftSubmission = {
   submission: {
     +[key: string]: mixed,
   },
   definition: Form,
+  previousFormSubmissionApprovalId: number | void,
```

Comment:
Perhaps I'm daft but why not previousFormSubmissionApprovalId?: number

Reply:
https://flow.org/en/docs/types/utilities/#toc-diff

i understand it as `previousFormSubmissionApprovalId?: number` means `previousFormSubmissionApprovalId` prop may or may not exist on the object , but `previousFormSubmissionApprovalId: number | void` will exist but be a number or undefined. using the former in `$Diff<>` wont actually remove the property while the latter will 🤷 

you can see the difference in action at this [flow playground thingy](https://flow.org/try/#0PTAEAEDMBsHsHcBQiAmBTAxtAhgJzaAC4CeADgQHJrwAiu2khAygK4BGAtgJYDOPXsAHagAvKADeiUKB7tufAYIBcEqdNABqANoBrNMRU9CuLoIDmAXRXcAHmhQAaNQF8n09JFNdCileK4oKoIsHGxouK5qpPgAbgIsPABisLgcrJy8-EIAgqTRsDHY0ACSgaDBoeGgAD6gMbABTs7I6Fh4BCTkoFTwyanp8lnCYgAkNFyQkAA8aj10DMxymYpuEtXRaHGwCX1pSwo5ebgFRaVBIWG4NXUNKDWRAHygAGSq0hjYpIQYABbYACqwPSCHiGYymMxaCxNZAYIRGUDYFQ9XYDZZCUQSD5fX4AoFoEEqLQAckgaDQkGJ0NAHi8PiEfgCKgAjK4ZPshn5nM1sAA6DZbHYpPYZA6CXL5QolFBAA)

Reply:
This should not be added here, It should be added to `NewFormsAppDraft`